packages/cli/src/commands/tile/deterministic.ts

```diff
@@ -19,16 +19,17 @@ export default class DeterministicTile extends Command<
     },
   ]
 
+  static flags = {
+    ...Command.flags,
+  }
   async run(): Promise<void> {
     this.spinner.start('Loading stream...')
 
     try {
       const tile = await TileDocument.deterministic(this.ceramic, this.args.metadata)
-      this.spinner.succeed(`Loaded stream ${tile.id.toString()}.`)
+      this.spinner.succeed(`Created tile ${tile.id.toString()}.`)
       this.logJSON({
         streamID: tile.id.toString(),
-        commitID: tile.commitId.toString(),
-        content: tile.content,
```

Comment:
I think having a dedicated `deterministic` command isn't very useful if it doesn't output the current content.

Reply:
oops, deleted a little too much. thanks for catching this.